content/en/docs/releases/next-release-preview/index.md

```diff
@@ -63,6 +63,24 @@ Addition of instance type metadata/ information to API response. See before-afte
   - The instance type info is also displayed for selected instance types like [this](https://user-images.githubusercontent.com/3614196/147279050-cc5837fa-0e02-4e17-b727-6b4a8baa6776.png).
 
 
+### Fiat
+- https://github.com/spinnaker/fiat/pull/972 introduces a way to control whether fiat queries clouddriver during a role sync.  The default is query.  To disable querying clouddriver for applications:
```

Comment:
incomplete statement: `the default is query`

Reply:
Thanks: https://github.com/spinnaker/spinnaker.io/pull/236/commits/a48f907b431529b9aa265e9ea1b43c08237e39cd